charts/openshift-metering/templates/presto/hive-metastore-statefulset.yaml

```diff
@@ -120,6 +120,14 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.presto.spec.hive.metastore.resources | indent 10 }}
+      - name: ghostunnel-server
```

Comment:
We should have a conditional around this new container section so that we can have it disabled by default, with an option to enable it.

Eg:

```
{{- if .Values.presto.spec.hive.config.tls.enabled }}
      - name: ghostunnel-server
        command: ["ghostunnel"]
....
{{- end }}
```

Reply:
Note, that this is required for CI to pass.